handsontable.d.ts

```diff
@@ -1862,10 +1862,10 @@ declare namespace Handsontable {
       beforeDrawBorders?: (corners: number[], borderClassName: 'current' | 'area' | 'highlight' | undefined) => void;
       beforeDropdownMenuSetItems?: (menuItems: contextMenu.MenuItemConfig[]) => void;
       beforeDropdownMenuShow?: (instance: plugins.DropdownMenu) => void;
-      beforeFilter?: (conditionsStack: plugins.FiltersPlugin.ColumnConditions[]) => void | boolean;
+      beforeFilter?: (conditionsStack: plugins.FiltersPlugin.ColumnConditions[]) => boolean;
```

Comment:
`beforeFilter` is not forcing boolean result, may not return anything. Keep `void`